pkgs/by-name/sd/SDL1/package.nix

```diff
@@ -143,13 +133,41 @@ stdenv.mkDerivation (finalAttrs: {
 
   setupHook = ./setup-hook.sh;
 
-  passthru = { inherit openglSupport; };
+  passthru = {
+    inherit openglSupport;
+
+    tests = lib.filterAttrs (key: value: value.meta.available) {
+      inherit
+        blender
+        directfb
+        hheretic
+        hhexen
+        mednafen
+        onscripter-en
+        powermanga
+        SDL_Pango
+        SDL_gfx
+        SDL_image
+        SDL_mixer
+        SDL_net
+        SDL_sound
+        SDL_ttf
+        sfxr
+        vlc
+        ;
+    };
+
+    updateScript = unstableGitUpdater {
+      tagPrefix = "release-";
+    };
+  };
 
   meta = {
     homepage = "http://www.libsdl.org/";
     description = "Cross-platform multimedia library";
     license = lib.licenses.lgpl21;
     mainProgram = "sdl-config";
+    changelog = "https://github.com/libsdl-org/SDL-1.2/blob/${finalAttrs.src.rev}/WhatsNew";
```

Comment:
is this stable with overrides?

Reply:
`rev` is set to `"refs/tags/${tag}"` if `tag` is instead used, if `src` is swapped for a `fetchurl` tarball then this would break. Luckily lazy-eval makes this for the most part a non-issue for downstream users